doc/user/filter.rst

```diff
@@ -165,14 +165,8 @@ Showing ip prefix-list
 Clear counter of ip prefix-list
 -------------------------------
 
-.. index:: clear ip prefix-list
-.. clicmd:: clear ip prefix-list
+.. index:: clear ip prefix-list [NAME [A.B.C.D/M]]
```

Comment:
shouldn't be `WORD`?

Reply:
it's the documentation not the internal cli.  I think NAME captures the better what we are looking for here.